tests/baselines/reference/privateNameStaticAccessors.js

```diff
@@ -0,0 +1,44 @@
+//// [privateNameStaticAccessors.ts]
+class A1 {
+    static get #prop() { return ""; }
```

Comment:
What if you had `static obj = new A1();` before this?

Reply:
Thank you for that comment, it resulted in a fix here for fields and two bugs for babel [this](https://github.com/babel/babel/issues/12905) and [this](https://github.com/babel/babel/issues/12904) 😊